hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStoreV1.java

```diff
@@ -216,6 +230,116 @@ public void testLookupKey() throws Exception {
             true);
   }
 
+  @Test
+  public void testListKeysWithNotNormalizedPath() throws Exception {
```

Comment:
Current test case only cover few cases of listKeysV1. Actually like startKey, maxKeys of listKeys are not tested. However, I also didn't find a good unit test added for original listKeys call.

Reply:
Yes, I will add more test cases. I'm thinking something like large dirs and sub-dirs/files.

Also, there is an open point: #listKeys("key-a-"), basically partial keyName and startKey. For example, actual keyName stored in DB are "key-a-10", "key-a-11", "key-a-12"..."key-z-10", "key-z-11", "key-z-12".
I will raise separate jira and work it on.